mifosng-android/src/main/java/com/mifos/mifosxdroid/online/noteitem/NoteItemFragment.java

```diff
@@ -0,0 +1,169 @@
+package com.mifos.mifosxdroid.online.noteitem;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.mifos.mifosxdroid.R;
+import com.mifos.mifosxdroid.adapters.NoteItemAdapter;
+import com.mifos.mifosxdroid.core.MifosBaseActivity;
+import com.mifos.mifosxdroid.core.MifosBaseFragment;
+import com.mifos.mifosxdroid.core.util.Toaster;
+import com.mifos.objects.noncore.NoteItem;
+import com.mifos.utils.Constants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by rahul on 4/3/17.
+ */
+public class NoteItemFragment extends MifosBaseFragment implements NoteItemMvpView,
+        SwipeRefreshLayout.OnRefreshListener {
+
+    @BindView(R.id.rv_note)
+    RecyclerView rvNote;
+
+    @BindView(R.id.progressbar_note)
+    ProgressBar pbNote;
```

Comment:
No need to add the ProgressBar in UI, Use MifosBaseFragment Dynamic ProgressBar.
You can access using the ` showMifosProgressBar();` and hide by `hideMifosProgressBar();`